db-scheduler/src/test/java/com/github/kagkarlsson/scheduler/task/CronTest.java

```diff
@@ -125,7 +128,28 @@ public void should_mark_schedule_as_disabled() {
   public void should_not_fail_on_cronschedule_without_next_execution_time() {
     CronSchedule cron =
         new CronSchedule("0 0 0 29 2 MON#1", ZoneId.systemDefault(), CronStyle.SPRING53);
-    Assertions.assertEquals(Schedule.NEVER, cron.getNextExecutionTime(complete(Instant.now())));
+    assertEquals(Schedule.NEVER, cron.getNextExecutionTime(complete(Instant.now())));
+  }
+
+  @Test
+  public void validate_cron_schedule_equals() {
+    assertEquals(
+        new CronSchedule("* * * * *", ZoneId.systemDefault(), CronStyle.UNIX),
+        new CronSchedule("* * * * *", ZoneId.systemDefault(), CronStyle.UNIX));
+    assertNotEquals(
+        new CronSchedule("1 * * * *", ZoneId.systemDefault(), CronStyle.UNIX),
+        new CronSchedule("* * * * *", ZoneId.systemDefault(), CronStyle.UNIX));
+    assertNotEquals(
+        new CronSchedule("1 * * * *", london, CronStyle.UNIX),
+        new CronSchedule("* * * * *", ZoneId.systemDefault(), CronStyle.UNIX));
```

Comment:
This one was prob not supposed to have variation in cron-pattern. (and maybe not use zone-default here, test will fail in London, no?)

Reply:
Yes my bad fixed it.